//test exception

int main()
{
    return 5 % 0;
}
